### PR TITLE
align with latest react-i13n code, add link scan for doc links

### PR DESCRIPTION
**Pull request by kaesonho**
align with latest react-i13n interface, also add the auto scan features, now we can track the links inside `dangerouslySetInnerHTML `
@redonkulus @mridgway @Vijar

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: components/Doc.jsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Doc extends React.Component {
         return (
             <div id="main" role="main" className="D(tbc)--sm Px(10px) Pos(r)">
                 {editEl}
-                <div onClick={this.onClick.bind(this)} dangerouslySetInnerHTML={{__html: markup}}></div>
+                <I13nDiv key={new Date().getTime()} onClick={this.onClick.bind(this)} dangerouslySetInnerHTML={{__html: markup}} scanLinks={{enable: true}}></I13nDiv>
```
Comment (kaesonho) [inline components/Doc.jsx]: add key with time here is for getting this div be forced `unmount` and `mount`, instead of reuse the same component.

**Comment by redonkulus** [comment]
:+1:

**Comment by Vijar** [comment]
:shipit:

**Comment by kaesonho** [comment]
can we get this merged?

**Pull request merged**